calico-enterprise/reference/architecture/overview.mdx

```diff
@@ -25,7 +25,7 @@ Calico open-source components are the foundation of {{prodname}}. {{prodname}} p
 
 - [calicoq](#calicoq)
 - [Compliance](#compliance)
-- [Elasticsearch gateway](#elasticsearch-gateway)
+- [Linseed API](#linseed-api)
```

Comment:
Should this be [Datastore API](#datastore-api)  ?

Reply:
Agreed upon Linseed API/ES gateway for this version.